lib/global_uid/error_tracker.rb

```diff
@@ -0,0 +1,7 @@
+module GlobalUid
+  class ErrorTracker
+    def notify(exception)
```

Comment:
We could also use the method name `#call` instead, a common trick to allow people to use a Proc/Lambda as well as a proper class.

Reply:
Good point. Updated.